recipes/yahmm/meta.yaml

```diff
@@ -33,7 +33,7 @@ test:
 
 about:
   home: http://pypi.python.org/pypi/yahmm/
-  license: LICENSE.txt
```

Comment:
Should this be license_file?

Reply:
could be , but needs to access source. Could be added as a link to source location (http:///), don't know how you usually manage references in source files.

Reply:
LICENSE.txt is indeed in the tarball, so you can change this to license_file:

Reply:
added license_info